test/e2e/suites/managed/helpers.go

```diff
@@ -51,6 +52,11 @@ const (
 	EKSControlPlaneOnlyLegacyFlavor                   = "eks-control-plane-only-legacy"
 )
 
+const (
```

Comment:
I assume this is because we don't have any contexts being passed into the helpers?

We should probably look into doing that at later date, but it's fine here for now.

Reply:
Yes, I had a look and it is quite a pain to do. I opted for these constants as I didn't want to put together a major refactor. I think it is acceptable for such trivial numeric consts.

Reply:
Yep, no objections to that. Just ensuring e2es are passing then I'll approve.